README.md

```diff
@@ -35,30 +40,36 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`EXPERIMENT_ID` : set the experiment ID, or leave unset for default experiment.
+`BIOMAGE_DATA_PATH`: where to get the experiment data to populate inframock's S3 and DynamoDB. If
+this is not set, it will default to `./data`.
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
-set to `true` by default, which means that InfraMock will try to use a locally running version of
-the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint.
-For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
-
-`MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the Github URL where the dataset used for the `worker` unit
-test is located.
+`POPULATE_MOCK`: whether localstack should be filled with the data sets found in `BIOMAGE_DATA_PATH`.
+For this to work, `CLUSTER_ENV` must be set to `development`, which is the default behavior.
 
 `AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set,
 it defaults to `eu-west-1`.
 
-`USE_LOCAL_DATA`: Flag to use local data or pull from Github. Set to `false` by defult. Set to `true` to use local data.
-
-Using local copy of data
+Adding custom data
 ---------------------
 
-Everytime Inframock is run, it will pull data from [](https://github.com/biomage-ltd/worker/tree/master/data/test). These files can be large. Inframock has `USE_LOCAL_DATA` environment variable path to command inframock to use local data. To use this feature :
+Inframock loads automatically the experiments found in the `./data` folder. The default experiment included is the same found in the worker repo [here](https://github.com/biomage-ltd/worker/blob/master/data/test/r.rds.gz). The expected format for loading data is the following:
+
+/data
+|-- e52b39624588791a7889e39c617f669e
+|   |-- mock_experiment.json
```

Comment:
It will be good to add in here that there is a schema for how to name the files:
    # * update DynamoDB checks for files matching mock_*.json
    # * update S3 checks for data files named r.rds.gz
    # * if the file is named in any other way, it will be ignored

Reply:
added it